crds/client.go

```diff
@@ -91,9 +91,9 @@ func (o *KubernetesClientOptions) Manager(namespace string, startCacheFor ...ctr
 			NewCache: func(_ *rest.Config, _ cache.Options) (cache.Cache, error) {
 				return &informertest.FakeInformers{}, nil
 			},
-			NewClient: func(_ cache.Cache, _ *rest.Config, _ ctrlruntimeclient.Options) (ctrlruntimeclient.Client, error) {
-				return fakectrlruntimeclient.NewFakeClient(), nil
-			},
+			// NewClient: func(_ cache.Cache, _ *rest.Config, _ ctrlruntimeclient.Options) (ctrlruntimeclient.Client, error) {
+			// 	return fakectrlruntimeclient.NewFakeClient(), nil
+			// },
```

Comment:
If someone can help me with that I will really appreciate it.

the `NewClient` does not exist anymore and the `ClientBuilder` is the new one. But I'm not sure how to deal with that.

Reply:
tried this
```
ClientBuilder: func(_ cache.Cache, _ *rest.Config, _ ctrlruntimeclient.Options) (ctrlruntimeclient.Client, error) {
    return fakectrlruntimeclient.NewClientBuilder().Build(), nil
},
```

but got 

```
cannot use (func(_ cache.Cache, _ *rest.Config, _ ctrlruntimeclient.Options) (ctrlruntimeclient.Client, error) literal) (value of type func(_ cache.Cache, _ *rest.Config, _ client.Options) (client.Client, error)) as cluster.ClientBuilder value in struct literal: missing method Build
```

Reply:
@alvaroaleman can you help with this?

Reply:
Seems that would currently require re-implementing the ClientBuilder, I'd recommend to leave the c-r bump out of this.

I've created a PR upstream to re-add this, but it's breaking so I doubt it will make it into an existing release: https://github.com/kubernetes-sigs/controller-runtime/pull/1409

Reply:
revert the bump